### PR TITLE
Add multiple screenshot unit test with some initialization Devtools configurations using CDP and WinForms.ChromiumWebBrowser

### PR DESCRIPTION
**Pull request by No5972**
Stack Overflow issue [https://stackoverflow.com/questions/65334430/message-id-went-wrong-when-using-cef-devtools-executedevtoolsmethodasync-and-pag#comment-117541474](https://stackoverflow.com/questions/65334430/message-id-went-wrong-when-using-cef-devtools-executedevtoolsmethodasync-and-pag#comment-117541474)

**Fixes:** [issue-number] 

**Summary:** [summary of the change and which issue is fixed here]
   - Add multiple screenshot unit test with some initialization Devtools configurations

**Changes:** [specify the structures changed] 
   - Modified ```DevToolsClientFacts```
      - Added a new Unit Test case
      - Multiple screenshot unit test with some initialization Devtools configurations
      
**How Has This Been Tested?**  
1. Instantiate a ```WinForms.ChromiumWebBrowser``` in a ```Form```
2. Do some initialization configurations of Devtools first, and then execute ```DevToolsExtensions.CaptureScreenShotAsPng```

**Screenshots (if appropriate):**
[](https://imgtu.com/i/6Qd4tx)

**Types of changes**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Updated documentation
- [x] Other: Test Case

**Checklist:**

- [x] Tested the code(if applicable)
- [x] Commented my code
- [ ] Changed the documentation(if applicable)
- [ ] New files have a license disclaimer
- [ ] The formatting is consistent with the project (project supports .editorconfig)

### CODE REVIEW

File: CefSharp.Test/DevTools/DevToolsClientFacts.cs [hunk 1]
```diff
@@ -156,5 +157,90 @@ public async Task CanUseMultipleDevToolsClientInstancesPerBrowser()
...
+
+                browser.LoadPageAsync();
+                browser.IsBrowserInitializedChanged += (sender, e) =>
+                {
+                    System.Diagnostics.Trace.WriteLine(browser.GetDevToolsClient().ToString());
+
+                    if (browser.IsBrowserInitialized)
+                    {
+                        output.WriteLine("Browser Initialized");
+                        Cef.UIThreadTaskFactory.StartNew(() =>
```
Comment (amaitland) [inline CefSharp.Test/DevTools/DevToolsClientFacts.cs:185]: Seems totally unrelated to testing DevTools, please remove.

File: CefSharp.Test/DevTools/DevToolsClientFacts.cs [hunk 2]
```diff
@@ -156,5 +157,90 @@ public async Task CanUseMultipleDevToolsClientInstancesPerBrowser()
...
+                        Application.DoEvents();
+                    }
+
+                    output.WriteLine(Cef.ChromiumVersion);
+
+                    var result = CefSharp.Example.DevTools.DevToolsExtensions.CaptureScreenShotAsPng(browser).Result;
+
+                    Assert.True(result.Length > 0, "Screenshot content is null!");
+                    output.WriteLine("Screenshot length: {0}", result.Length);
+                    MessageBox.Show(result.Length.ToString());
```
Comment (amaitland) [inline CefSharp.Test/DevTools/DevToolsClientFacts.cs:218]: Please remove all instances of MessageBox.Show, automated builds hang when attempting to run this test.

**Review by amaitland — CHANGES_REQUESTED** [review]
What is the actual stack trace you are getting?

File: CefSharp.Test/DevTools/DevToolsClientFacts.cs (hunk 1 above)
Comment (No5972) [reply CefSharp.Test/DevTools/DevToolsClientFacts.cs]: ```requestContext``` code is removed

File: CefSharp.Test/DevTools/DevToolsClientFacts.cs (hunk 2 above)
Comment (No5972) [reply CefSharp.Test/DevTools/DevToolsClientFacts.cs]: ```MessageBox``` code is removed

**Comment by No5972** [comment]
> What is the actual stack trace you are getting?

The actual stack trace does not displayed which it only displays "One or more errors occurred".

When using "Debug" in the Test Explorer, the actual stack trace is still ```null```. The message is "Generated MessageId 2 doesn't match returned Message Id 1".

**Comment by amaitland** [comment]
Build was successful,  test passed.

https://ci.appveyor.com/project/cefsharp/cefsharp/build/job/jp88in09w7wc2aew#L623

**Comment by No5972** [comment]
Can't reproduce on server...

**Pull request closed without merging**